include/rocksdb/iterator.h

```diff
@@ -53,11 +54,13 @@ class Iterator : public Cleanable {
   // All Seek*() methods clear any error status() that the iterator had prior to
   // the call; after the seek, status() indicates only the error (if any) that
   // happened during the seek, not any past errors.
+  // Target does not contain timestamp.
```

Comment:
I don't see a similar comment to InternalIterator. I think we should comment the same there.

Reply:
Sure, will add a comment. Note that internal iterator seek target does contain user timestamp.